cpp/tensorrt_llm/thop/cublasScaledMM.cpp

```diff
@@ -130,12 +69,13 @@ bool find_special_algo(cublasLtMatmulAlgo_t& algo, std::shared_ptr<CublasMMWrapp
         && compType == CUBLAS_COMPUTE_32F)
     {
         // TODO: remove this after cublas fix the heuristic for Spark
-        algo_list = tensorrt_llm::common::getSMVersion(/*queryRealSmArch=*/true) == 121 ? spark_bf16_algo_list
-                                                                                        : bf16_algo_list;
+        algo_list = tensorrt_llm::common::getSMVersion(/*queryRealSmArch=*/true) == 121
```

Comment:
Existing issue + probably doesn't matter much but this is going to cause a copy

Reply:
good call, I switched to pointers